loaddictionaries.cc

```diff
@@ -117,7 +117,7 @@ void LoadDictionaries::handlePath( Config::Path const & path )
   for( QFileInfoList::const_iterator i = entries.constBegin();
        i != entries.constEnd(); ++i )
   {
-    QString fullName = i->canonicalFilePath();
+    QString fullName = i->filePath();
```

Comment:
[`QFileInfo::absoluteFilePath()`](https://doc.qt.io/qt-5/qfileinfo.html#absolutePath) should also fix the issue and is closer to `canonicalFilePath()`, so it is probably less risky than `filePath()`.

Reply:
I agree. goldendict seems always assume dict paths are absolute. Although the `QFileInfo` here should always be absolute.
I rebased and force-updated my fork to use `absoluteFilePath()` now.